examples/migrate_forseti/tutorial.md

```diff
@@ -0,0 +1,336 @@
+  - Security Reviewer (`roles/iam.securityReviewer`)
+  - Organization Policy Viewer (`roles/orgpolicy.policyViewer`)
+  - Sevice Management Quota Viewer (`roles/servicemanagement.quotaViewer`)
+  - Service Usage Consumer (`roles/serviceusage.serviceUsageConsumer`)
+  - Compute Security Admin (`roles/compute.securityAdmin`)
+  - Storage Object Viewer (`roles/storage.objectViewer`)
+  - Storage Object Creator (`roles/storage.objectCreator`)
+  - CloudSQL Client (`roles/cloudsql.client`)
+  - Stackdriver Log Writer (`roles/logging.logWriter`)
+  - Service Account Token Creator (`roles/iam.serviceAccountTokenCreator`)
+
+#### Updated In-Place
+
+- The `forseti-client-deny-all-RESOURCE_NAME_SUFFIX` firewall rule and
+  the `forseti-server-deny-all-RESOURCE_NAME_SUFFIX` firewall rule will
+  both update from denying all protocols to denying ICMP, TCP, and UDP
+- The `forseti-server-allow-grpc-RESOURCE_NAME_SUFFIX` firewall rule
+  will update to only allow traffic from the
+  `forseti-client-gcp-RESOURCE_NAME_SUFFIX` service account and to allow
+  traffic to port 50052 in addition to 50051
+- The `forseti-client-RESOURCE_NAME_SUFFIX` and `forseti-server-RESOURCE_NAME_SUFFIX` 
+  GCS bucket to remove bucket labels
+- The `forseti-cai-export-RESOURCE_NAME_SUFFIX` GCS bucket to remove bucket
+  labels and update lifecycle conditions
+- The `forseti-client-gcp-RESOURCE_NAME_SUFFIX` and `forseti-server-gcp-RESOURCE_NAME_SUFFIX` 
+  service accounts will be updated in place to change the display name
+- The `forseti-server-db-RESOURCE_NAME_SUFFIX` CloudSQL database will
+  increase in resource size.  It will also gain the `net_write_timeout`
+  flag.
+  
+
+#### Destroyed and Replaced
+
+- The `forseti-client-allow-ssh-external-RESOURCE_NAME_SUFFIX` firewall
+  rule and the `forseti-server-allow-ssh-external-RESOURCE_NAME_SUFFIX`
+  firewall rule will both be replaced due to a naming change, but the new firewall
+  rules will be equivalent
+- The `forseti-client-vm-RESOURCE_NAME_SUFFIX` compute instance and the
+  `forseti-server-vm-RESOURCE_NAME_SUFFIX` compute instance will be
+  replaced due to changes in metadata startup scripts, boot disk sizes
+  and boot disk types; these VMs should be stateless but ensure that
+  any customizations are captured before applying this change
+- The `configs/forseti_conf_client.yaml` object in the
+  `forseti-client-RESOURCE_NAME_SUFFIX` storage bucket and the
+  `configs/forseti_conf_server.yaml` object in the
+  `forseti-server-RESOURCE_NAME_SUFFIX` storage bucket will be replaced
+  due to a lack of Terraform import support
+
+## Apply the Terraform Changes
+Execute the following to apply the Terraform plan.
+```sh
+terraform apply
+```
+
+At this point, the existing Forseti deployment has been migrated to a
+Terraform state.
+
+## Save state to GCS
+Congratulations, you have now installed Forseti!
+
+As a final step, you will want to save your configuration so it can be used to upgrade Forseti in the future.
```

Comment:
Please add the word "important":
```
IMPORTANT: As a final step, you will want to save your configuration so it can be used to upgrade Forseti in the future.
```

Reply:
ack